integration-tests/realm-apps/pv-int-tests/services/mongodb/rules/defaultRole.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Just want to double check, we're not able to remove the permission rule in `services/mongodb/config.json`? Might be the case that it's only allowed in flexible sync configurations but thought it'd be worth to ask.

Reply:
Correct, it only applies to `flexible sync`.  I attempted to remove these permissions for partition based as well and the API returned an error saying the required fields were missing.